aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -30,19 +29,21 @@ At the end of the series, you'll have an app that manages and displays movie dat
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-mvc-app/start-mvc/sample) ([how to download](xref:index#how-to-download-a-sample)).
 
+At the end of the series, you'll have an app that manages, validates and displays movie data. 
+
+![Completed application showing the search UI](~/tutorials/first-mvc-app/search/_static/9/search2_90.png)
+
+![Completed application showing client side valiation features](~/tutorials/first-mvc-app/validation/_static/9/val90.png)
```

Comment:
Consider omitting this; at this point in the tutorial it is a big block of clutter to have to scroll through.